src/components/Trade/TradeInput.tsx

```diff
@@ -284,10 +289,22 @@ export const TradeInput = () => {
     setTradeAmountsRefetchData,
     buyAsset?.assetId,
   ])
-
   const onSubmit = useCallback(async () => {
     setIsLoading(true)
     try {
+      if (sellAsset && buyAsset && mixpanel) {
+        const compositeBuyAsset = getMaybeCompositeAssetSymbol(buyAsset.assetId, assets)
+        const compositeSellAsset = getMaybeCompositeAssetSymbol(sellAsset.assetId, assets)
+        mixpanel.track(MixPanelEvents.TradePreview, {
+          buyAsset: compositeBuyAsset,
+          sellAsset: compositeSellAsset,
+          fiatSellAmount,
+          fiatBuyAmount,
```

Comment:
This one actually tracks`fiatSellAmount` and `fiatBuyAmount` correctly, looks like a mismatch between this one and preview step?